setup.py

```diff
@@ -23,36 +23,31 @@
 from setuptools import Extension, setup
 from setuptools.command.build_ext import build_ext
 
-IS_WINDOWS = sys.platform == 'win32'
-IS_APPLE = sys.platform == 'darwin'
-IS_X64 = platform.architecture()[0] == '64bit'
+IS_WINDOWS = sys.platform == "win32"
+IS_APPLE = sys.platform == "darwin"
+IS_X64 = platform.architecture()[0] == "64bit"
 
 if IS_WINDOWS:
     # https://stackoverflow.com/a/57109148/6639989
     import distutils.cygwinccompiler
+
     distutils.cygwinccompiler.get_msvcr = lambda: []
 
-with open('README.md', 'r') as fh:
+with open("README.md", "r") as fh:
     readme = fh.read()
-    long_description = '\n'.join(readme.split('\n')[2:]).lstrip()
+    long_description = "\n".join(readme.split("\n")[2:]).lstrip()
 
 # Throughout this file "mgclient" can mean two different things:
 # 1. The mgclient library which is the official Memgraph client library.
 # 2. The mgclient python extension module which is a wrapper around the
 #    client library.
-EXTENSION_NAME = 'mgclient'
+EXTENSION_NAME = "mgclient"
 
-sources = [
-    'column.c', 'connection-int.c', 'connection.c', 'cursor.c',
-    'glue.c', 'mgclientmodule.c', 'types.c'
-]
-sources = [os.path.join('src', fn) for fn in sources]
+sources = ["column.c", "connection-int.c", "connection.c", "cursor.c", "glue.c", "mgclientmodule.c", "types.c"]
```

Comment:
Consider using glob, since you are including all *.c files (fun fact glob from pathlib is os independant :) )
```suggestion
from pathlib Path
sources = Path("src").glob("*.c")
```

Reply:
I opted in for explicit listing, as I prefer that over implicit ways (like glob). I don't have any really strong reasons why, so if you think it is better I can switch it. My reasons are:
 1. in CMake globbing has some disadvantages, python and `setup.py` might also have some issues with it.
 2. Postgres [did the same for psycopg](https://github.com/psycopg/psycopg2/blob/c34bf2f2f90fbbe65e8c4c2650734aa151ddd6ed/setup.py#L456-L475)

What do you think?

Reply:
The advantage I see is that when using glob you don't have to touch the setup when adding new files since in most cases you want to use all src/hdr files. But currently, it's not a big deal for me you can leave it as is.

Reply:
You are the relative python expert, I trust your word!